examples/views/new_components.py

```diff
@@ -0,0 +1,81 @@
+from io import BytesIO
+
+from discord import (
+    ApplicationContext,
+    Bot,
+    ButtonStyle,
+    Color,
+    File,
+    Interaction,
+    SeparatorSpacingSize,
+    User,
+)

```

Comment:
The other examples use a different import style for these classes. I think we should use that instead.